test/json_unittest.cc

```diff
@@ -0,0 +1,385 @@
+#include <functional>
+
+#include "../src/json.h"
+#include "gtest/gtest.h"
+
+#define EXPECT_TO_STRING(v, s) EXPECT_EQ(v->ToString(), s)
```

Comment:
The convention seems to be `EXPECT_EQ(expected, actual)`.

Reply:
Done.